docs/installation/environment-variables.md

```diff
@@ -142,87 +142,10 @@ Configure the following environment variable to enable MicroPowerManager to send
-| `METER_DATA_URL`     | **Required** (when plugin is used) | Calin Meter API URL used for consumption data upload.      |
-| `METER_DATA_KEY`     | **Required** (when plugin is used) | Calin Meter API key used for consumption data upload.      |
-| `METER_DATA_USER`    | **Required** (when plugin is used) | Calin Meter API username used for consumption data upload. |
-
-#### Calin Smart Meters
-
-For detailed information see [Calin Smart Meter Developer Documentation](https://www.szcalinmeter.com/).
-
-| Environment Variable           | Default                            | Description                                                               |
-| ------------------------------ | ---------------------------------- | ------------------------------------------------------------------------- |
-| `CALIN_SMART_COMPANY_NAME`     | **Required** (when plugin is used) | Calin Smart Meter company name used for communication with Calin API.     |
-| `CALIN_SMART_PURCHASE_API_URL` | **Required** (when plugin is used) | Calin Smart Meter Purchase API URL used for communication with Calin API. |
-| `CALIN_SMART_CLEAR_API_URL`    | **Required** (when plugin is used) | Calin Smart Meter Clear API URL used for communication with Calin API.    |
-| `CALIN_SMART_USER_NAME`        | **Required** (when plugin is used) | Calin Smart Meter username used for communication with Calin API.         |
-| `CALIN_SMART_PASSWORD`         | **Required** (when plugin is used) | Calin Smart Meter password used for communication with Calin API.         |
-| `CALIN_SMART_PASSWORD_VENT`    | **Required** (when plugin is used) | Calin Smart Meter password vent used for communication with Calin API.    |
-
-#### Vodacom
-
-Vodacom integration requires a VPN tunnel with Vodacom infrastructure.
-For detailed information see [Vodacom Developer Documentation](https://business.m-pesa.com/vodacom-tanzania/business-onboarding-tanzania/)
-
-| Environment Variable            | Default                            | Description                 |
-| ------------------------------- | ---------------------------------- | --------------------------- |
-| `VODACOM_SPID`                  | **Required** (when plugin is used) | Vodacom SPID.               |
-| `VODACOM_SPPASSWORD`            | **Required** (when plugin is used) | Vodacom SPPASSWORD.         |
-| `VODACOM_IPS`                   | **Required** (when plugin is used) | Vodacom IPs.                |
-| `VODACOM_REQUEST_URL`           | **Required** (when plugin is used) | Vodacom Request API URL.    |
-| `VODACOM_BROKER_CRT`            | **Required** (when plugin is used) | Path to broker `.crt` file. |
-| `VODACOM_SLL_KEY`               | **Required** (when plugin is used) | Path to `.key` file.        |
-| `VODACOM_CERTIFICATE_AUTHORITY` | **Required** (when plugin is used) | Path to `.cer` file.        |
-| `VODACOM_SSL_CERT`              | **Required** (when plugin is used) | Path to `.pem` file.        |
```

Comment:
These seem used

- https://github.com/EnAccess/micropowermanager/blob/main/src/backend/config/services.php#L22-L33
- https://github.com/EnAccess/micropowermanager/blob/main/src/backend/app/modules/Transaction/Provider/VodacomTransactionProvider.php#L311-L323

Please don't remove them

Reply:
VodacomTransactionProvider should be removed. We need to develop a plugin for Airtel If we're planning that in future.

Reply:
Resolved here: https://github.com/EnAccess/micropowermanager/pull/526